### PR TITLE
Fixed Win Server 2019 node definition and added NXOSv 9500

### PR DESCRIPTION
**Pull request by CML-TAC**
Changes
Fixed Win Server 2019 node definition.
Added NXOSv 9500 with 128 ports
Deleted duplicate readme for palo alto

### CODE REVIEW

File: node-definitions/cisco/NXOSv 9500 128 ports/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# NXOSv 9500 with 128 ports
+This directory contains the following node definition:
+
+*`nxosv9500-148.yaml` - Nexus 9500v with 128 ports node definition
```
Comment (xorrkaz) [inline node-definitions/cisco/NXOSv 9500 128 ports/readme.md:4]: This says 128 switch ports, but your node def talks about 148.  Ultimately, I do see that it is 148 (64 + 16 + 32 + 36).  So it's probably just this typo that needs to be fixed.

**Review by xorrkaz — CHANGES_REQUESTED** [review]
See comment.

File: node-definitions/cisco/NXOSv 9500 128 ports/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# NXOSv 9500 with 128 ports
```
Comment (tmikuska) [inline node-definitions/cisco/NXOSv 9500 128 ports/readme.md:1]:
```suggestion
# NXOSv 9500 with 148 ports
```

Comment (tmikuska) [reply node-definitions/cisco/NXOSv 9500 128 ports/readme.md:1]: I thought that I would be able to apply a suggestion but no permissions here.

**Review by xorrkaz — APPROVED** [review]
Looks good.

**Pull request merged**